src/tests/ftest/container/delete.py

```diff
@@ -22,37 +22,22 @@
   portions thereof marked with this legend must also reproduce the markings.
 """
 
-import json
 import os
-import sys
 import time
 import traceback
 import uuid
 
-sys.path.append('./util')
-sys.path.append('../util')
-sys.path.append('../../../utils/py')
-sys.path.append('./../../utils/py')
+from apricot import TestWithServers
 
-import server_utils
-import write_host_file
-import AgentUtils
-from avocado import Test
-from daos_api import DaosApiError, DaosContainer, DaosContext, DaosLog, DaosPool
+from daos_api import DaosPool, DaosApiError
 
-class DeleteContainerTest(Test):
+class DeleteContainerTest(TestWithServers):
     """
     Tests DAOS container delete and close.
+    :avocado: recursive
     """
     def setUp(self):
-        self.agent_sessions = None
-        # get paths from the build_vars generated by build
-        self.hostlist = None
-        with open('../../../.build_vars.json') as build_file:
-            build_paths = json.load(build_file)
-        self.basepath = os.path.normpath(build_paths['PREFIX'] + "/../")
-        self.server_group = self.params.get("server_group", '/server/',
-                                            'daos_server')
+        super(DeleteContainerTest, self).setUp()
 
         # parameters used in pool create
         self.createmode = self.params.get("mode",
```

Comment:
confused, this is staying?  Why?

Reply:
Because the base classes don't (currently) do any fetching of the `mode` (or `size`, or other attributes used sporadically across the tests) from the params as it's not clear that this is universal across all tests for a given base class.

We may decide on a set of params that should be used for all tests and implement them consistently in the base class and tests in the future.  But I didn't want this patch to be changing the substance of the tests as they exist currently.